wolfcrypt/src/asn.c

```diff
@@ -1633,6 +1633,10 @@ static const byte extExtKeyUsageEmailProtectOid[] = {43, 6, 1, 5, 5, 7, 3, 4};
 static const byte extExtKeyUsageTimestampOid[]    = {43, 6, 1, 5, 5, 7, 3, 8};
 static const byte extExtKeyUsageOcspSignOid[]     = {43, 6, 1, 5, 5, 7, 3, 9};
 
+#ifdef WOLFSSL_OID_IPSEC_IKE
```

Comment:
We typically don't disable a specific extension OID like this is doing here (WOLFSSL_OID_IPSEC_IKE).  I'm not sure it's very user friendly to make the user realize they need to define a specific define in order to enable support for an key usage OID (which they may not even realize existed).  It may be better to simply enable this by default, then wolfSSL will be able to process this extension OID by default?

Reply:
Eliminated